tests/install_testbed/change_idm_password.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
```

Comment:
Delete this line is is not a component that could be executed directly from the CLI.